rke/structure_rke_cluster_network.go

```diff
@@ -162,6 +162,10 @@ func flattenRKEClusterNetwork(in rancher.NetworkConfig) []interface{} {
 		obj["aci_network_provider"] = flattenRKEClusterNetworkAci(in.AciNetworkProvider)
 	}
 
+  if in.EnableBrNetfilter != nil {
+		obj["enable_br_netfilter"] = in.EnableBrNetfilter
```

Comment:
The field needs to be handled in the corresponding expander function.

Reply:
done